wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,380 @@
+    height: 173px;
+  }
+  .figure-space {
+    max-width: 265px;
+    position: relative;
+    z-index: -9;
+  }
+  .figure-logo {
+    max-width: 150px;
+    position: relative;
+    z-index: -9;
+  }
+  .body {
+    fill: #6DDCBD;
+    -webkit-transform: scaleX(1.25);
+    transform: scaleX(1.25);
+    -webkit-transform-origin: center;
+    transform-origin: center;
+    stroke-width: 3px;
+  }
+  .box {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+  .circle {
+    fill: #6DDCBD;
+    stroke: #fff;
+    stroke-width: 7px;
+  }
+  h2 {
+    font-size: 22px;
+    max-width: 500px;
+    margin: 5px auto 0;
+  }
+  main p {
+    font-size: 16px;
+    line-height: 20px;
+    max-width: 390px;
+    margin: 15px auto 0;
+    color: #000000;
+  }
+  footer {
+    padding: 25px 0;
+    position: fixed;
+    left: 50%;
+    bottom: 0;
+    width: 960px;
+    -webkit-transform: translateX(-50%);
+    transform: translateX(-50%);
+    -webkit-transform-style: preserve-3d;
+    transform-style: preserve-3d;
+    border-top: 1px solid #efefef;
+  }
+  .option {
+    float: left;
+    width: 33.33%;
+    box-sizing: border-box;
+    padding-right: 5px;
+  }
+  .option svg {
+    width: 25px;
+    height: 25px;
+    fill: gray;
+    border: 1px solid #d6d6d6;
+    padding: 5px;
+    border-radius: 100%;
+    float: left;
+    margin-right: 10px;
+  }
+  .option div {
+    display: table;
+  }
+  .option h4 {
+    color: #246060;
+    font-size: 19px;
+  }
+  .option h4:hover {
+    text-decoration: underline;
+  }
+  .option p {
+    font-weight: 300;
+    font-size: 15px;
+    padding-top: 3px;
+    color: #000000;
+  }
+  @media (max-width: 996px) {
+    body, footer {
+      max-width: 780px;
+    }
+  }
+  @media (max-width: 800px) {
+    footer, main {
+      position: relative;
+    }
+    footer {
+      height: 100%;
+    }
+    .option {
+      position: relative;
+      width: 100%;
+      left: auto;
+      right: auto;
+      top: auto;
+      padding: 0 25px;
+      margin-bottom: 60px;
+    }
+    .two {
+      margin-left: 0px;
+      -webkit-transform: none;
+      transform: none;
+    }
+    .option {
+      float: none;
+    }
+    .figure {
+      margin-top: 10px;
+    }
+    main {
+      padding: 0 25px;
+    }
+    main h2 {
+      font-size: 18px;
+    }
+    main p {
+      font-size: 14px;
+    }
+    header {
+      padding-left: 20px;
+      padding-right: 20px;
+    }
+    footer {
+      overflow: hidden;
+      width: 100%;
+      margin-top: 50px;
+    }
+  }
+  @media (min-width: 801px) and (max-height: 730px) {
+    .figure {
+      margin-top: 80px;
+    }
+  }
+  @media (min-width: 801px) and (max-height: 600px) {
+    footer {
+      position: relative;
+      margin: 135px auto 0;
+    }
+    .figure {
+      margin-top: 50px;
+    }
+  }
+</style>
+<header>
+  <div class="row">
+    <div class="column">
+      <div class="logo">
+        <a href="https://wagtail.io/" target="_blank" rel="noopener">
```

Comment:
Should be `rel="nofollower noopener"` for browsers that do not support noopener (IE11, Edge).

Reply:
Fixed.